### PR TITLE
example multi agents

### PR DESCRIPTION
**Pull request by matarrese**

### CODE REVIEW

File: examples/MultiAgentsLog2Kafka/src/main/resources/qiao.xml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+            <property name="clientId" value="${emitter.kafka.clientId}" />
+            <property name="acknowledgePolicy" value="${emitter.kafka.ackPolicy}" />
+            <property name="serializerClassname" value="kafka.serializer.StringEncoder" />
+            <property name="keySerializerClassname" value="com.aol.advertising.qiao.emitter.kafka.LongSerializer" />
+            <property name="messageHandler" ref="stringEventHandler" />
+         </emitter>
+
+      </funnel>
+      <property name="fileManager" ref="qiaoFileManager" />
+      <property name="fileBookKeeper" ref="fileBookKeeper" />
```
Comment (ytung05) [inline examples/MultiAgentsLog2Kafka/src/main/resources/qiao.xml]: Valid property names for agent are 'bookKeeper' and 'fileManager'.

File: examples/MultiAgentsLog2Kafka/src/main/resources/qiao.xml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+      <funnel id="FileFunnel" rate-limit="${FilesFunnel-rate-limit}" auto-start="${FilesFunnel-autostart}" qsize="${FilesFunnel-dataPipeCapacity}">
+
+         <injector id="fileInjector" class="com.aol.advertising.qiao.injector.PatternMatchFileInjector"
+                   initPositionsFrom="tailInjector">
+            <property name="readMode" value="${fileinjector.readmode}" />
+            <property name="dataHandler" ref="binDataHandler" />
+            <property name="srcDir" value="${fileinjector.srcDir}" />
+            <property name="filePattern" value="${fileinjector.filePattern}" />
+            <property name="doneFileHandler" ref="doneFileHandler" />
```
Comment (ytung05) [inline examples/MultiAgentsLog2Kafka/src/main/resources/qiao.xml]: Remove doneFileHandler and fileBookKeeper.  They are injected internally by the bootstrapping process.

File: examples/MultiAgentsLog2Kafka/src/main/resources/qiao.xml [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+      <funnel id="TailFunnel" auto-start="${LogTailer-autostart}" emitter-threads="${LogTailer-emitter-threads}">
+
+         <injector id="tailInjector" class="com.aol.advertising.qiao.injector.TailInjector">
+            <property name="filename" value="${tailinjector.filename}" />
+            <property name="tailMode" value="${tailinjector.tailmode}" />
+            <property name="dataHandler" ref="binDataHandler" />
+            <property name="bufSize" value="${tailinjector.readBufferSize}" type="int" />
+            <property name="checksumByteLength" value="${tailinjector.checksumByteLength}" type="int" />
+            <property name="cacheDefaultExpirySecs" value="${tailInjector.positionCache.defaultExpirySecs}" type="int" />
+            <property name="fileBookKeeper" ref="fileBookKeeper" />
```
Comment (ytung05) [inline examples/MultiAgentsLog2Kafka/src/main/resources/qiao.xml]: Remove fileBookKeeper.  BookKeeper is injected internally.

File: examples/MultiAgentsLog2Kafka/src/main/resources/beans.xml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+http://www.springframework.org/schema/jms http://www.springframework.org/schema/jms/spring-jms.xsd 
+http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context.xsd 
+http://www.springframework.org/schema/task http://www.springframework.org/schema/task/spring-task.xsd 
+http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd">
+
+   <bean id="binDataHandler" class="com.aol.advertising.qiao.example.log2kafka.MyBinDataHandler" scope="prototype" />
+
+   <bean id="stringEventHandler" class="com.aol.advertising.qiao.emitter.handler.DefaultKafkaStringMessageHandler" scope="prototype" />
+
+   <bean id="fileManager-2" class="com.aol.advertising.qiao.injector.file.watcher.QiaoFileManager">
```
Comment (ytung05) [inline examples/MultiAgentsLog2Kafka/src/main/resources/beans.xml]: See Examples/Log2Kafka for example.  No need to define doneFileHandler bean.

**Comment by matarrese** [comment]
@ytung05 please review this new updates, thanks

**Review by ytung05 — CHANGES_REQUESTED** [review]
Please remove all the files related to qiao_cache.

**Comment by matarrese** [comment]
Hi @ytung05 I saw there are some cached files as well in the folder Log2Kafka on this branch , should they be removed as well from there?

**Comment by ytung05** [comment]
The qiao_cache directory in Log2Kafka has been removed.  PR #6 has been merged.  Thanks.

**Pull request merged**